src/Razor/Microsoft.NET.Sdk.Razor/integrationtests/Wasm/WasmBuildIntegrationTest.cs

```diff
@@ -55,10 +59,14 @@ public async Task Build_InRelease_Works()
             Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "blazor.boot.json");
             Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "blazor.webassembly.js");
             Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "dotnet.wasm");
+            Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "dotnet.wasm.gz");
             Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", DotNetJsFileName);
             Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "blazorwasm.dll");
```

Comment:
Out of interest, why aren't we checking for `blazorwasm.dll.gz` and `"RazorClassLibrary.dll.gz`? Are we only precompressing the BCL?

Reply:
Also in the other test above

Reply:
Right, I see below that we are only compressing the BCL